runtime/appruntime/metrics/README.md

```diff
@@ -0,0 +1,16 @@
+# `metrics`
+
+This package allows an Encore app to export custom metrics to cloud providers.
+
+We have two types of custom metrics: a predefined set of custom metrics which are enabled for all Encore apps and
+user-defined custom metrics. User-defined custom metrics aren't supported yet.
+
+## Predefined Encore metrics
+
+These are the metrics all Encore apps export to cloud providers:
+
+- `e_requests_total` measures the number of requests and has two tags `service` and `endpoint`.
```

Comment:
I'd change the prefix to `encore_` and then reserve that prefix

Reply:
I rather like that the prefix is short and sweet. Seems unlikely to collide with anything?

Reply:
We've gone with the short prefix.

Reply:
Are you sure that no user ever will want to use `e_` as a prefix in their structured logs?

I can easily imagine people wanting to use `enc_` as a prefix, what's the issue with using `encore_`? Given that it's not actually visible to users at the end?